cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -146,6 +146,17 @@ def _validate(self, instance_type: str, image: str):
                     ),
                     FailureLevel.ERROR,
                 )
+        unsupported = ["p3", "p2", "g3", "g2"]
+        if (
+            image_info
+            and "AWS ParallelCluster AMI" in image_info.description
+            and instance_type.split(".")[0] in unsupported
+        ):
+            self._add_failure(
+                f"The instance type '{instance_type}' is not supported by OpenRM drivers. "
```

Comment:
The instance type '{instance_type}' is not supported by NVIDIA OpenRM drivers. Only Ampere, Turing, and Hopper architectures are supported. Please consider using a different instance type or building a custom AMI with closed source NVIDIA drivers.

Reply:
We could create a constant for supported architectures too, like:
`NVIDIA_OPENRM_SUPPORTED_ARCHS = "Ampere, Turing, and Hopper"`